src/app/hooks/useToggle/README.md

```diff
@@ -1,9 +1,8 @@
 # useToggle Hook
 
-## Feature Toggles
+### Feature Toggles
 
-Feature toggles are designed to be global (i.e not service specific) and are dependant on the application environment.
-Feature toggles can be found in `src/app/lib/config/toggles`
+A description of how feature toggles work can be found in `/src/app/lib/config/toggles/README.md`
```

Comment:
can we make that a link to https://github.com/bbc/simorgh/blob/latest/src/app/lib/README.md

Reply:
Yes, absolutely. Done!